### PR TITLE
Fix broken field keys query in FieldKeyType

### LINKED ISSUE #76: Field key query failing
When creating measurement field inside a form, the current query to get `fieldKey` seem to be broken
cf : https://github.com/Algatux/influxdb-bundle/blob/master/src/Form/Type/FieldKeyType.php#L26

Then we get this error : 
```
unable to use system source '_fieldKeys': use SHOW FIELD KEYS instead
```

### PR DESCRIPTION
**Pull request by Jerome1337**
Fixes #76 

Use correct syntax for querying field keys

### CODE REVIEW

**Comment by Algatux** [comment]
Hi @Jerome1337, thanks for the PR, we will review it asap.

**Comment by soullivaneuh** [comment]
@Algatux I'll checkout the fork and try it on my project ASAP.

**Comment by Algatux** [comment]
@Soullivaneuh thx a lot!

File: src/Form/Type/FieldKeyType.php [hunk 1]
```diff
@@ -22,10 +22,7 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'choices' => function (Options $options) {
                     return $this->loadChoicesFromQuery(
-                        sprintf(
-                            'SELECT fieldKey FROM "default"._fieldKeys WHERE _name = \'%s\'',
-                            $options['measurement']
-                        ),
+                        sprintf('SHOW FIELD KEYS FROM "%s"', $options['measurement']),
```
Comment (soullivaneuh) [inline src/Form/Type/FieldKeyType.php]: Please keep the previous sprintf multi-lines to have a clearer diff.

**Review by soullivaneuh — CHANGES_REQUESTED** [review]
This indeed fix the issue with the latest versions of InfluxDB.

I left 1 request and Travis need to be fixed.

**Comment by Jerome1337** [comment]
@Algatux @Soullivaneuh I modified sprintf to multi-line. FYI the solution was found here : https://docs.influxdata.com/influxdb/v0.9/query_language/schema_exploration/#explore-field-keys-with-show-field-keys

**Comment by soullivaneuh** [comment]
@Jerome1337 Travis is still Failing.

**Comment by soullivaneuh** [comment]
Thank you @Jerome1337 !

**Pull request merged**